config/locales/en/activemodel.yml

```diff
@@ -509,14 +509,14 @@ en:
               blank_with_partner: Enter how much the home your client and their partner lives in is worth
               greater_than_or_equal_to: How much the home your client lives in in must be 0 or more
               greater_than_or_equal_to_with_partner: How much the home your client and their partner lives in must be 0 or more
-              not_a_number: How much the home your client lives in must be an amount of money, like 60,000
+              not_a_number: Enter the amount the home is worth, like 1,000 or 20.30
```

Comment:
I know this is exactly what's on the spreadsheet, but a home worth 20.30?  Seems a bit odd!